<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGameInputDialogue</class>
 <widget class="QDialog" name="UserGameInputDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="callCheckRButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>112</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Call</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="raiseRButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>112</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Raise</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="foldRButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>112</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Fold</string>
   </property>
  </widget>
  <widget class="QLabel" name="cashLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cash:</string>
   </property>
  </widget>
  <widget class="QLabel" name="raiseByLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Raise By:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="raiseEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserGameInputDialogue</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserGameInputDialogue</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
